InternetShop/src/main/java/dao/OrderDaoFileImpl.java

```diff
@@ -0,0 +1,78 @@
+package dao;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import model.Order;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class OrderDaoFileImpl implements OrderDao {
+    private ObjectMapper mapper = new ObjectMapper();
+    private File file = new File("InternetShop/src/main/java/resource/orders.json");
+    private Map<Long, Order> ordersMap = new LinkedHashMap<>();
+
+    @Override
+    public List<Order> getAllOrders() {
+        readWithFile();
+        return new ArrayList<>(ordersMap.values());
+    }
+
+    @Override
+    public void createOrderInDatabase(Order order) {
+        readWithFile();
+        ordersMap.put(order.getId(), order);
+        writeIntoFile();
```

Comment:
same for write into file, in most cases you want to write into it on any operation. basically, its a flush operation:

https://stackoverflow.com/questions/3220336/whats-the-use-of-session-flush-in-hibernate